rfs/src/store/mod.rs

```diff
@@ -16,21 +17,32 @@ pub async fn make<U: AsRef<str>>(u: U) -> Result<Stores> {
     let parsed = url::Url::parse(u.as_ref())?;
 
     match parsed.scheme() {
-        dir::SCHEME => return Ok(Stores::Dir(
-            dir::DirStore::make(&u)
-                .await
-                .expect("failed to make dir store"),
-        )),
-        "s3" | "s3s" | "s3s+tls" => return Ok(Stores::S3(
-            s3store::S3Store::make(&u)
-                .await
-                .expect(format!("failed to make {} store", parsed.scheme()).as_str()),
-        )),
-        "zdb" => return Ok(Stores::ZDB(
-            zdb::ZdbStore::make(&u)
-                .await
-                .expect("failed to make zdb store"),
-        )),
+        dir::SCHEME => {
+            return Ok(Stores::Dir(
+                dir::DirStore::make(&u)
+                    .await
+                    .expect("failed to make dir store"),
+            ))
+        }
+        "s3" | "s3s" | "s3s+tls" => {
+            return Ok(Stores::S3(s3store::S3Store::make(&u).await.expect(
+                format!("failed to make {} store", parsed.scheme()).as_str(),
+            )))
+        }
+        "zdb" => {
+            return Ok(Stores::ZDB(
+                zdb::ZdbStore::make(&u)
+                    .await
+                    .expect("failed to make zdb store"),
+            ))
+        }
+        "http" | "https" => {
+            return Ok(Stores::HTTP(
+                http::HTTPStore::make(&u)
+                    .await
+                    .expect("failed to make http store"),
```

Comment:
I know you just following the code that exists but why do we use expect() here and panic instead of returning a proper error. I think this was just a shortcut that got committed upstream. Please replace all expect in this function with a proper error